package-lock.json

```diff
@@ -7499,7 +7500,8 @@
           "version": "2.1.1",
           "resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-2.1.1.tgz",
           "integrity": "sha1-w7M6te42DYbg5ijwRorn7yfWVN8=",
-          "dev": true
+          "dev": true,
+          "optional": true
```

Comment:
Do you know what all these "optional" additions to the package-lock mean?

Reply:
searching around in google seems to suggest that came from me install from newer version of npm
[Stack Overflow](https://stackoverflow.com/questions/54411377/package-lock-json-file-package-with-optional-true)

Reply:
should i remove them?

Reply:
Fine to keep them, we can revert if there are any issues!